components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -635,9 +635,10 @@ void AdsServiceImpl::OnInitialize(
 
     // OnBraveAdsServiceReady is implemented for Android for now
     #if defined(OS_ANDROID)
-      auto* display_service_impl =
-          static_cast<NotificationDisplayServiceImpl*>(display_service_);
-      display_service_impl->OnBraveAdsServiceReady(is_initialized_);
+// TODO(jocelyn): FIXME
+//      auto* display_service_impl =
+//          static_cast<NotificationDisplayServiceImpl*>(display_service_);
+//      display_service_impl->OnBraveAdsServiceReady(is_initialized_);
```

Comment:
This part should be addressed by https://github.com/brave/brave-browser/issues/6813.